config/installation_config.yml

```diff
@@ -126,6 +126,26 @@
   type: boolean
 # ------- End of Facebook Channel Related Config ------- #
 
+# ------- WhatsApp Channel Related Config ------- #
```

Comment:
@tds-1 Let's use Installation Config instead of environment variables so we can update values through the Superadmin interface. Reference https://github.com/chatwoot/chatwoot/pull/11548